test/shopify-cli/shopifolk_test.rb

```diff
@@ -0,0 +1,46 @@
+require 'test_helper'
+
+module ShopifyCli
+  class ShopifolkTest < MiniTest::Test
+    GCLOUD_FEATURE_NAME = 'gcloud_shopifolk'
+    DEV_FEATURE_NAME = 'dev_shopifolk'
+
+    def test_correct_features_is_shopifolk
+      gcloud_path = '../shopifolk_correct.conf'
+      ShopifyCli::Feature.disable(GCLOUD_FEATURE_NAME.to_s)
```

Comment:
Why do we need to call `.to_s` on the constants? Seems they are already strings.

Reply:
😅 my bad, copied it over from another test file where they set their constants like this `TEST_FEATURE = :feature_set_flag_test`, is it better form for me to change my constant declaration than to remove the `.to_s`?

Reply:
Using strings for these constants is fine, not sure why the other uses a symbol.